### DISCUSSION TITLE
MakerFocus 1.3inch TFT LCD Display - Pale, washed out colors

### DISCUSSION THREAD

**Discussion by hai8825** [Bodmer/TFT_eSPI#3146] [Q&A - General]
Hello , I have the following [display](https://www.makerfocus.com/products/1-3inch-tft-lcd-display-module-3-3v-with-spi-interface-st7789-ic-driver?variant=31337673064525) that is using the ST7789 driver. However when I upload an image to display the image is too bright and the colors are kinda washed out/crushed. Like there is not enough contrast, and there is a bit of distortion. Are there any settings in the TFT_eSPI Drivers folder or the User Setups that can be tweaked to make the colors appear normal as I see them on my desktop. Attached is an image of what it kinds seems but it is worst in person.
![IMG_9906](https://github.com/Bodmer/TFT_eSPI/assets/78314751/f9d795cf-4e0a-4256-93c1-dfc179ba2841)
Any help would be greatly appreciated.

**Comment by BDhackers009** [comment]
Sir which board do you have? I have esp32-s3 wroom-1 (n16r8) i couldn't get it working except powering it

**Reply by hai8825** [reply]
I have ESP8266 D1 Mini. I think I was able to get it on D1mini esp32. Have to remember what the wiring was.
